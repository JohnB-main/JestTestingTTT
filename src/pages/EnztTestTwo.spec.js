import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
 
import Board from './game.js';
 
describe('game', () => {
  describe('Reducer', () => {
  });
 
  test('snapshot renders', () => {
    const component = renderer.create(<game />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
 
});
 
