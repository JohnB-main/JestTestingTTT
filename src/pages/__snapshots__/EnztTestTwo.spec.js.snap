// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`game snapshot renders 1`] = `<game />`;
